Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-video bundle size difference from 6.0.3 to ba14ae444d094b84c7210311d6764a2049891be6
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-video bundle size difference from 6.0.3 to d16ca5c0f35dfcb33d3e84859069890182cd5fe1
No significant bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
